docs/website/contents/for-developers/EraTransitionGovernance.md

```diff
@@ -0,0 +1,140 @@
+# Era Transition Goverance
+
+This document explains how on-chain governance ends each era within Cardano, with a particular emphasis on testnets.
+
+Every era ends when on-chain governance increments the major protocol version past the greatest within that era.
+According to [the table maintained alongside CIP-59](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0059/feature-table.md), those versions are as follows.
+
+| Era | Least major version | Greatest major version |
+| - | - | - |
+| Byron | 0 | 1 |
+| Shelley | 2 | 2 |
+| Allegra | 3 | 3 |
+| Mary | 4 | 4 |
+| Alonzo | 5 | 6 |
+| Babbage | 7 | 8 |
+| Conway | 9 | >= 10 (Conway is the latest era at time of writing) |
+
+All eras forbid proposals to increase the major protocol version by more than one at a time.
+Thus the governance process within those eras (Byron, Alonzo, Babbage, and Conway, so far) that contain multiple major protocol versions must take place multiple times.
+In turn, for example, at least the first two epochs of any Cardano chain will be in the Byron era.
```

Comment:
> at least the first two epochs of any Cardano chain will be in the Byron era

Skipping to a later era is mentioned below, but maybe explicitly call this out here already so people are not confused? Especially as the first sentence above says

> with a particular emphasis on testnets.